### ISSUE TITLE
Unable to view JSON documents on Github (Blocked script execution)

### ISSUE THREAD

**Issue by benc-uk** [tulios/json-viewer#187]
Viewing any JSON documents on Github results in this error 

**"Blocked script execution in 'https://raw.githubusercontent.com/benc-uk/azure-arm/master/paas-web/basic-webapp/azuredeploy.json' because the document's frame is sandboxed and the 'allow-scripts' permission is not set."**

This is an example file https://raw.githubusercontent.com/benc-uk/azure-arm/master/paas-web/basic-webapp/azuredeploy.json 
and this https://raw.githubusercontent.com/benc-uk/microservices-demoapp/master/package.json

However **every** JSON file I've tried results in the same error, seems to only be an issue when viewing from Github (the raw view)

**Comment by LightDiscord** [comment]
Same for me I'm on Chromium 63

**Comment by jchappell82** [comment]
Same here on Chrome 63.0.3239.132

**Comment by leftler** [comment]
Also having the same problem. Chrome for Windows 64.0.3282.186 (Official Build) (64-bit)

**Comment by benc-uk** [comment]
I don't think this is getting fixed, I've switched to using another extension. Shame as this was really nice

**Comment by ffxsam** [comment]
@benc-uk Which extension did you wind up using?

**Comment by benc-uk** [comment]
This one seems ok - https://github.com/gildas-lormeau/JSONView-for-Chrome

Pretty basic but it lets you apply your own CSS to style the JSON. A few others I looked at were even more basic or didn't work

**Comment by LightDiscord** [comment]
It's weird but you can see the JSON you can't see on GitHub by copying it and paste it into the search bar like this: 'json-viewer <json>'
![json-viewer](https://user-images.githubusercontent.com/24509182/36944488-b2104d80-1f9d-11e8-8747-996572659e15.png)

**Comment by mrchief** [comment]
I made a dark theme gist for [gildas-lormeau/JSONView-for-Chrome](https://github.com/gildas-lormeau/JSONView-for-Chrome) that matches json-viewer styles. It's not as extensive (limitation of the other extension) as this extension but gets as close as possible.

[JSONView.dark.theme.css](https://gist.github.com/mrchief/4d3fa3aa19b31262e57e92146f440eb4)

**Comment by VladimirAlexiev** [comment]
https://github.com/tulios/json-viewer/pull/209. If @tulios doesn't merge it soon, I'm switching too.

**Comment by ngyikp** [comment]
FYI this is fixed on Chrome 68
https://github.com/tulios/json-viewer/issues/178#issuecomment-406840323
